main/zoe/guide/contract-installation-and-instance.md

```diff
@@ -0,0 +1,98 @@
+# Contract installation and instance
+
+Zoe has two methods to "create a contract": 
+- `zoe.install(code, moduleFormat)` which returns an `installationHandle`
+- `zoe.makeInstance(installationHandle, issuerKeywordRecord, terms)` which returns an `invite` to participate in the contract
+
+This two-step process to create may feel unnecessarily convoluted at first.
+
+The reason Zoe is designed this way is that it is assumed that the JavaScript source `code` of the contract may be large (several kilobytes, maybe several megabytes) and for the purpose of transparency, this source code will be stored in blockchains when Zoe runs on a blockchain.
+
+```js
+const mySuperContractSourceCode = ` ... 1 Megabyte string ... `;
+const moduleFormat = 'getExport';
```

Comment:
We should probably explain how to use `bundleSource` here, so that users don't think they have to do this themselves, and so that `moduleFormat` makes a little more sense.

```
import bundleSource from '@agoric/bundle-source';
const { source, moduleFormat } = await bundleSource(contractPath);
```